scripts/clearProperties.ts

```diff
@@ -0,0 +1,89 @@
+import {akamaiRequest} from "./utils/akamaiRequest";
+
+const filterProperties = async () => {
+    const {body: {properties: {items}}} = await akamaiRequest({
+        path: `/papi/v1/properties?contractId=${process.env.AK_CONTRACT_ID}&groupId=${process.env.AK_GROUP_ID}`,
+        method: 'GET',
+    });
+
+    const testDomain = new URL(process.env.TEST_DOMAIN ?? '');
+    const [_, hostname, tld] = testDomain.hostname.split('.');
+    const domain = `${hostname}.${tld}`;
+
+    return items.filter(t => t.propertyName.endsWith(domain)).filter(t => t.propertyName !== testDomain.hostname);
+}
+
+const checkDNSRecordExists = async (domain: string) => {
+    const response = await fetch(`https://api.cloudflare.com/client/v4/zones/${process.env.CF_ZONE_ID}/dns_records?name=${domain}`, {
+        headers: {
+            Authorization: `Bearer ${process.env.CF_AUTH_TOKEN}`,
+        }
+    })
+    const body = await response.json();
+
+    return body.result_info.total_count > 0;
+}
+
+const getActiveVersions = async (propertyId: string) => {
+    const {body: {activations: {items}}} = await akamaiRequest({
+        path: `/papi/v1/properties/${propertyId}/activations?contractId=${process.env.AK_CONTRACT_ID}&groupId=${process.env.AK_GROUP_ID}`,
+        method: 'GET',
+    });
+
+    return items.filter(t => t.status === 'ACTIVE' && t.activationType === 'ACTIVATE');
+}
+
+const deActivateVersion = async (propertyId: string, {network, version}: {
+    network: string,
+    version: number
+}) => {
+    return akamaiRequest({
+        path: `/papi/v1/properties/${propertyId}/activations?contractId=${process.env.AK_CONTRACT_ID}&groupId=${process.env.AK_GROUP_ID}`,
+        method: 'POST',
+        body: JSON.stringify({
+            network,
+            propertyVersion: version,
+            activationType: "DEACTIVATE",
+            notifyEmails: ['support+akamai@fingerprint.com'],
```

Comment:
is it a good idea to spam our support email? I also noticed that we used `support+akamai-ci@fingerprint.com` for activating properties.

Reply:
maybe we can use Orkun's and Necip's email addresses.

Reply:
Sharing people's company emails on a public repository didn't make sense to me. Maybe we can change it to support+akamai+ci WDYT?

Reply:
We can keep it in GH secrets. support email is for people to reach our support, and they will get spammed by Akamai, I don't think we should use support+anything at all.

Reply:
Having our emails spammed instead didn't make sense tho. Let's continue discussing this on slack.

Reply:
We decided to have an email address in secrets.